crates/sqlexec/src/planner/physical_plan/client_send.rs

```diff
@@ -151,17 +157,26 @@ struct ClientExchangeSendStream {
     /// record batch stream), and producing a stream of ipc encoded data, we
     /// don't really have a decent way forwarding errors through the stream.
     result: Arc<Mutex<ClientExchangeSendResult>>,
+
+    // needed to create an empty reocrd batch in case of a empty stream
+    schema: Arc<Schema>,
```

Comment:
Do we need this? Could we do `self.stream.schema()` in `poll_next`?

Reply:
yeah, we could do that. fixing